environment

```diff
@@ -104,33 +114,35 @@ DSS_AWS_FLASHFLOOD_PREFIX_READ_PROD="aws"
 DSS_AWS_FLASHFLOOD_PREFIX_WRITE_PROD="aws"  # This can be a comma separated list
 DSS_GCP_FLASHFLOOD_PREFIX_READ_PROD="gcp"
 DSS_GCP_FLASHFLOOD_PREFIX_WRITE_PROD="gcp"  # This can be a comma separated list
-DSS_INFRA_TAG_PROJECT=dcp
-DSS_INFRA_TAG_SERVICE=dss
-DSS_INFRA_TAG_OWNER=dss-team@data.humancellatlas.org
-DSS_NOTIFICATION_SENDER="DSS Notifier <dss.humancellatlas@gmail.com>"
-ADMIN_USER_EMAILS=""
-DCP_DOMAIN=${DSS_DEPLOYMENT_STAGE}.data.humancellatlas.org
-API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
+
+DSS_INFRA_TAG_PROJECT="cedar"
```

Comment:
i think this value should be dbio, maybe we can set this to whatever `${PLATFORM}` is if we use the comment above.

Reply:
Setting this to `${DSS_PLATFORM}-dss` which becomes `ucsc-cgp-dss`

LMK if you have a different idea in mind

Reply:
implemented with 36b3eb0